posts/2023-02-21-23.0.0.2-beta.adoc

```diff
@@ -0,0 +1,389 @@
+    <httpEndpoint id="defaultHttpEndpoint" httpPort="9080" httpsPort="9443"/>
+
+    <library id="derby">
+      <file name="${server.config.dir}/derby/derby.jar"/>
+    </library>
+
+    <dataSource id="DefaultDataSource">
+      <jdbcDriver libraryRef="derby"/>
+      <!-- Example properties referencing an in-memory Derby Embedded database -->
+      <properties.derby.embedded databaseName="memory:defaultdb" createDatabase="create"/>
+    </dataSource>
+
+    <authData id="myAuth" user="dbuser" password="dbpass"/>
+
+    <!-- Default SSL configuration enables trust for default certificates from the Java runtime --> 
+    <ssl id="defaultSSLConfig" trustDefaultCerts="true"/>
+
+    <remoteFileAccess>
+       <writeDir>${server.config.dir}</writeDir>
+    </remoteFileAccess>
+
+    <basicRegistry id="basic">
+       <user name="admin" password="adminpwd"/>
+    </basicRegistry>
+
+    <!-- Assign 'admin' to Administrator -->
+    <administrator-role>
+        <user>admin</user>
+    </administrator-role>
+
+</server>
+----
+
+For this `server.xml` example, you will need to add the derby jar to the server configuration or use your own database configuration.
+
+1. Configure a Liberty server using the above `server.xml` for guidance and then start the server. Once started, you can check the logs to find the URL to navigate to the Admin Center. In the above example, you would navigate to the Admin Center using https://localhost:9443/adminCenter/ URL.
+
+2. Select "Server Config" tool.
++
+[.img_border_light]
+image::img/blog/ac_db_test_server_config.png[Server Config Tool,width=20%,align="center"]
+
+3. Select "server.xml" to edit.
++
+[.img_border_light]
+image::img/blog/ac_db_test_serverxml.png[server.xml,width=50%,align="center"]
+
+4. Navigate to the resource you would like to test and click the "Test" button.
++
+[.img_border_light]
+image::img/blog/ac_db_test_resource.png[Select resource,width=50%,align="center"]
+
+5. Choose which type of authentication your application uses:
++
+* For applications that use container authentication: 
++
+For this example, the config doesn't have default authentication specified on the datasource element nor are there any login modules configured, therefore you will need to specify an authentication alias by using the dropdown.
++
+[.img_border_light]
+image::img/blog/ac_db_test_container_auth.png[Container authentication,width=50%,align="center"]
+
+
+* For applications that use application authentication, choose the "Application authentication" tab and fill in a valid user name and password for the database resource.
++
+[.img_border_light]
+image::img/blog/ac_db_test_app_auth.png[Application authentication,width=50%,align="center"]
+
+* If your application does not use a resource reference and you have not `enableContainerAuthForDirectLookups="true"` in the config element, then choose "No resource reference" tab and fill in a valid user name and password for the database resource.

```

Comment:
Is this missing a word? Should it instead be,
"... and you have not specified `enableContainerAuthForDirectLookups="true"` ..."

Reply:
```suggestion
* If your application does not use a resource reference and the server.xml doesn't include `enableContainerAuthForDirectLookups="true"` in the config element, then choose "No resource reference" tab and fill in a valid user name and password for the database resource.

```